aiokafka/conn.py

```diff
@@ -216,7 +233,11 @@ def _do_version_lookup(self):
 
     @asyncio.coroutine
     def _do_sasl_handshake(self):
-        req_klass = self._version_info.pick_best(SaslHandShakeRequest)
+        if self._sasl_mechanism == "GSSAPI":
+            # v1 is unsupported, trigger a infinite auth loop
```

Comment:
I feel like this is not true. The v1 is supported, we just need to figure out how it's done. It works with Java client )

Reply:
dunno, both kafka-python and rdkafka use v0

Reply:
Lol, version v1 did not work for such a dumb reason. Basically a few of my buggy lines of code where req_klass is used in 2 places and next request just always ended up with API_VERSION=0 =). It works for both versions, kafka-python and rdkafka just did not pick this up yet.